src/custom/pages/Profile/index.tsx

```diff
@@ -83,14 +69,14 @@ export default function Profile({ ENSName }: AccountDetailsProps) {
                 <span role="img" aria-label="farmer">
                   🧑‍🌾
                 </span>
-                <strong>-</strong>
+                <strong>{profileData?.totalTrades?.toLocaleString() || '-'}</strong>
```

Comment:
`toLocaleString`?

Also, this is an integer, do you need to convert to string?

Reply:
The one and only reason to use `toLocaleString` was the thousands separator: `1,000` instead of `1000`

Reply:
Oh I see! Good idea